<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialog</class>
 <widget class="QDialog" name="FindReplaceDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>208</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string extracomment="Find and replace text with this dialog.">Find &amp; Replace</string>
  </property>
  <property name="windowOpacity">
   <double>0.750000000000000</double>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4">
    <widget class="QPushButton" name="findButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="findAndReplace">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Find and replace</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="caseSensitive">
     <property name="text">
      <string>Case sensitive</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" rowspan="2">
    <widget class="QPushButton" name="replaceAllButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Replace all</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QRadioButton" name="findDownwards">
     <property name="text">
      <string>Find downwards</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QRadioButton" name="findUpwards">
     <property name="text">
      <string>Find upwards</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>String to replace:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>String to be found:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="findText">
     <property name="text">
      <string extracomment="Text to find"/>
     </property>
     <property name="placeholderText">
      <string>Text to find</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="replaceText">
     <property name="text">
      <string extracomment="Replacing text"/>
     </property>
     <property name="placeholderText">
      <string>Replacing text</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="doneButon">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="walkAround">
     <property name="text">
      <string>Walk around the document</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="useRegExp">
     <property name="text">
      <string>Use regular expression</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="completeWordsOnly">
     <property name="text">
      <string>Complete words only</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findText</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>replaceText</tabstop>
  <tabstop>findAndReplace</tabstop>
  <tabstop>replaceAllButton</tabstop>
  <tabstop>caseSensitive</tabstop>
  <tabstop>useRegExp</tabstop>
  <tabstop>walkAround</tabstop>
  <tabstop>completeWordsOnly</tabstop>
  <tabstop>findDownwards</tabstop>
  <tabstop>findUpwards</tabstop>
  <tabstop>doneButon</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
